op-acceptance-tests/tests/base/advance_test.go

```diff
@@ -29,3 +30,11 @@ func TestCLAdvance(gt *testing.T) {
 		return new_num > num
 	}, 30*time.Second, waitTime)
 }
+
+// TestCLAdvance_Supernode runs the same advance test using the supernode L2CL implementation.
+func TestCLAdvance_Supernode(t *testing.T) {
+	prev := os.Getenv("DEVSTACK_L2CL_KIND")
+	_ = os.Setenv("DEVSTACK_L2CL_KIND", "supernode")
+	defer func() { _ = os.Setenv("DEVSTACK_L2CL_KIND", prev) }()
```

Comment:
I like how you're wrapped the existing test!

Note that there's a built-in for the temporary environment variable setting.
It'll make our implementation shorter and probably also safer for parallel runs.

```suggestion
	t.Setenv("DEVSTACK_L2CL_KIND", "supernode")
```